pages/index.jsx

```diff
@@ -2,6 +2,7 @@ import Head from 'next/head'
 import { AppLayout, Placeholder } from '@/components'
 import Banner from '@/components/Banner/Banner'
 import labels from '@/constants/labels.en'
+import Hero from '@/components/Hero/Hero'
```

Comment:
Please add Hero component to components/index and import it as the others on the line 2

Reply:
Added to components/index.js